.github/workflows/main.yml

```diff
@@ -16,6 +16,7 @@ jobs:
     name: Linting Check
     uses: the-guild-org/shared-config/.github/workflows/lint.yml@main
     with:
+      packageManagerVersion: 4.2.2
```

Comment:
I'm not happy with this option because if `yarn` were to be upgraded, the version number would have to be bumped in a few places. A nicer solution would be to add `corepack` support to `the-guild-org/shared-config/.github/workflows`.